app/models/ministry.rb

```diff
@@ -28,8 +28,7 @@ class Ministry < ActiveRecord::Base
 
   has_many :user_content_locales, dependent: :destroy
   has_many :stories, dependent: :destroy
-  has_many :ministry_areas, dependent: :destroy
-  has_many :areas, through: :ministry_areas
+  belongs_to :area, dependent: :destroy
```

Comment:
Remove the dependent destroy, from rails guides:
"You should not specify this option on a belongs_to association that is connected with a has_many association on the other class. Doing so can lead to orphaned records in your database."